index.bs

```diff
@@ -547,17 +547,85 @@ navigator.getVRDisplays().then(function (displays) {
 </pre>
 </div>
 
+## VRDisplayEventReason ## {#interface-vrdisplayeventreason}
+
+<pre class="idl">
+interface VRDisplayEventReason {
+  readonly attribute unsigned short code;
+  const unsigned short VR_DISPLAY_NAVIGATION = 1;
+  const unsigned short VR_DISPLAY_MOUNTED = 2;
+  const unsigned short VR_DISPLAY_UNMOUNTED = 3;
+};
+</pre>
+
+### Attributes ### {#vrdisplayeventreason-attributes}
+
+<dfn attribute for="VRDisplayEventReason">code</dfn>
+The code describing the reason for this event, described below.
+
+### Reason Codes ### {#vrdisplayeventreason-codes}
+
+<dfn attribute for="VRDisplayEventReason">VR_DISPLAY_NAVIGATION</dfn>
+The page has been navigated to from a context, such as another site that was already in VR presentation mode, that allows this page to begin presenting immediately.
+
+<dfn attribute for="VRDisplayEventReason">VR_DISPLAY_MOUNTED</dfn>
+The {{VRDisplay}} has detected that the user has put it on.
+
+<dfn attribute for="VRDisplayEventReason">VR_DISPLAY_UNMOUNTED</dfn>
+The {{VRDisplay}} has detected that the user has taken it off.
+
+## VRDisplayEvent ## {#interface-vrdisplayevent}
+
+<pre class="idl">
+[Constructor(VRDisplayEventInit eventInitDict)]
+interface VRDisplayEvent : Event {
+  readonly attribute VRDisplay display;
+  readonly attribute VRDisplayEventReason? reason;
+};
+
+dictionary VRDisplayEventInit : EventInit {
+    required VRDisplay display;
+};
+</pre>
+
+### Attributes ### {#vrdisplayevent-attributes}
+
+<dfn attribute for="VRDisplayEvent">display</dfn>
+The {{VRDisplay}} associated with this event.
+
+<dfn attribute for="VRDisplayEvent">reason</dfn>
+{{VRDisplayEventReason}} describing why this event has has been fired.
+
 
 ## Window Interface extension ## {#interface-window}
 
 <pre class="idl">
 partial interface Window {
   attribute EventHandler onvrdisplayconnected;
   attribute EventHandler onvrdisplaydisconnected;
+  attribute EventHandler onvrdisplayactivate;
```

Comment:
Good point. I'll update to "activated" and "deactivated". (Side note: I actually had a bug in my code over the weekend where I had assumed that it was past tense in one spot but not the other. Consistency for the win!)